packages/tasks-gen/snippets-fixtures/automatic-speech-recognition/js/huggingface.js/0.hf-inference.js

```diff
@@ -0,0 +1,13 @@
+import { InferenceClient } from "@huggingface/inference";
+
+const client = new InferenceClient("api_token");
+
+const data = fs.readFileSync("sample1.flac");
+
+const output = await client.automaticSpeechRecognition({
```

Comment:
More general remark: the types for stand-alone methods (`automaticSpeechRecognition`) have a correct typing, while methods on the `InferenceClient` class (`client.automaticSpeechRecognition`) do not

Until this is fixed, I would advocate to use the stand-alone functions for better user experience

Reply:
opened https://github.com/huggingface/huggingface.js/issues/1294 as a follow-up issue